.github/workflows/gh-pages.yml

```
@@ -23,6 +23,11 @@ jobs:
         with:
           command: install
           args: --debug --path ./forc
+      - name: Install Forc plugins
+        uses: actions-rs/cargo@v1
+        with:
+          command: install
+          args: forc-fmt forc-lsp forc-explore
```

If we don't need release builds here (don't think we do), then debug builds will be faster.
```suggestion
          args: --debug forc-fmt forc-lsp forc-explore
```

---

that makes sense - thanks for the catch!